meilisearch-release-changelog/v0.30.0.md

```diff
@@ -0,0 +1,229 @@
+# v0.30.0 release changelogs
+
+<!-- The following line should NOT be put in the official release changelogs -->
+⚠️ Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+This release removes index deployment downtime and adds new task management features. It also introduces new search parameters that facilitate the creation of numbered pagination interfaces.
+
+Thanks to Hacktoberfest, tons of updates and bug fixes have been done by the fantastic Meilisearch community. You were more present than ever and words cannot make justice tojust  how grateful we are for your support. A big thank you from the whole Meilisearch team! ❤️
+
+<!-- The following line should NOT be put in the official release changelogs -->
+📖 Meilisearch's official documentation will be ready for the official v0.30.0 release. Meanwhile, work-in-progress documentation is available [here]().
+
+# New features and improvements 🔥
+
+## Ease search results navigation with pagination
+
+You can now navigate through page selection when searching with Meilisearch using the API. This way, you will get the exact total number of pages and results Meilisearch has matched based on a query.
+
+For this, use `page` and `hitsPerPage` as query parameter instead of `offset` and `limit`:
+
+```bash
+curl \
+  -X POST 'http://localhost:7700/indexes/movies/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "shifu", "page": "2", "hitsPerPage": "10" }'
+```
+
+```json
+{
+    "hits": [
+        // … 10 hits
+    ],
+    // …
+    "page": 2,
+    "hitsPerPage": 10,
+    "totalHits": 2100,
+    "totalPages": 210
+}
+```
+
+Done in #2601 by @ManyTheFish.
+
+## Cancel processing and enqueued tasks
+
+Meilisearch now allows you to cancel `enqueued` or `processing` tasks through a new API route, `/tasks/cancel`. This can be useful when you need to interrupt a process that is taking too much time to be completed.
+
+To cancel a task, select it by specifying one of the following query parameters: `uid`, `status`, `type`, or `indexUid`. You can also select a task by date using one of the following query parameters: `afterEnqueuedAt`, `afterStartedAt`, `afterFinishedAt`, `beforeEnqueuedAt`,`beforeStartedAt`, or `beforeFinishedAt`.
```

Comment:
```suggestion
To cancel tasks, filters them by specifying one of the following query parameters: `uid`, `status`, `type`, or `indexUid`. 

Tasks can be filtered by date fields: 
- `beforeEnqueuedAt` / `afterEnqueuedAt`
- `beforeStartedAt` / `afterStartedAt`
- `beforeFinishedAt` / `afterFinishedAt`

```

Reply:
I understand that "filter" is technically accurate, but I'm not sure it's helpful in this context. "filter the task you want to delete" sounds like a much more complex concept than "select the task you want to delete".